pkgs/applications/misc/bambu-studio/default.nix

```diff
@@ -166,6 +166,12 @@ stdenv.mkDerivation rec {
     )
   '';
 
+  # needed to prevent collisions between the LICENSE.txt files of
+  # bambu-studio and orca-sliver.
+  postInstall = ''
+    mv $out/LICENSE.txt $out/LICENSE-${finalAttrs.pname}.txt
```

Comment:
i think this is ok, an other candidate location could be `$out/share/doc/` and there is also `$out/README.md` that may deserve similar treatment

I'll leave it upto you, unless others have suggestions

Reply:
I like the idea of using `$out/share/doc/`! i will change it this evening once I find time.
Thank you for the input!